src/main/resources/feedbackQuestionTextEditFormTemplate.html

```diff
@@ -0,0 +1,10 @@
+<div>
+    <br>
+    <div class="row">
+        <div class="col-sm-4" data-toggle="tooltip" data-placement="top" title="recommended length ">Recommended Length:
```

Comment:
Also I feel that the tooltip wouldn't be necessary as it is a duplicate of the text